features/car/access.feature

```diff
@@ -305,3 +305,16 @@ Feature: Car - Restricted access
             | primary    | psv          |       |
             | primary    | no           |       |
             | primary    | customers    |   x   |
+
+    Scenario: Car - Conditional restrictions
+        Then routability should be
+            | highway      | motor_vehicle   |  motor_vehicle:conditional            | bothw |
+            | primary      | yes             |                                       |   x   |
+            | primary      | yes             |  no @ 2002 Jan 7 - 2002 Feb 8         |   x   |
+            | primary      | yes             |  no @ 2002 Jan 07 - 2002 Feb 08       |   x   |
+            | primary      | yes             |  no @ 2090 Jan 7 - 2100 Feb 8         |   x   |
+            | primary      | yes             |  no @ 2020 Jan 7 - 2050 Feb 8         |       |
```

Comment:
Will this test break in 24 years? 😉

Reply:
Yep 😄 I can extend the ranges so they don't break that soon, but I don't see any simple way to get completely rid of this problem.

Reply:
Ah, this can be solved doing something similar to [this](https://github.com/ivarbrek/osrm-backend/blob/bf2ca0a4ee51b0c993039eda4b9e641c5fb5032e/features/car/conditional_restrictions.feature#L42-L48). I'll try to do that.